lib/src/authorization_code_grant.dart

```diff
@@ -100,6 +109,9 @@ class AuthorizationCodeGrant {
   /// The current state of the grant object.
   _State _state = _State.initial;
 
+  /// The generated PKCE code verifier.
+  String _codeVerifier;
```

Comment:
Hmm, given that this object is already stateful I suspect this somewhat okay... But this does make it very hard to use on a server since you can't store the code verifier in a database/cache... Maybe we should consider this..

Reply:
Maybe we should take this as an argument?

Also we should document how to provide this... How to cache it...